content/article/cardano-cube-find-every-project-thats-building-on-the-cardano-blockchain.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Cardano Cube - Find every project that's building on the Cardano Blockchain
+tags:
+  - platform
+  - Ecosystem
+  - dAppStore
+url: ""
+image: https://ucarecdn.com/8a643b14-d54f-486f-90ed-bef4dcde2618/
+image_text: Cardano Ecosystem Map
+---
+
+It’s no secret that cryptocurrency Cardano is growing every day. With a massive amount of ADA staked by it’s investors, it looks like a promising cryptocurrency for the long-term. Many people in the cryptocurrency community even label the project as “The blockchain of the future.”
```

Comment:
'With a massive amount of ADA staked by it’s investors, it looks like a promising cryptocurrency for the long-term.'
Please change the following:
- ADA to ada (we usually don't capitalize it according to the style guide)
- it's to its